### ISSUE TITLE
Add a time-ago widget?

### ISSUE THREAD

**Issue by ckcollab** [RiotGear/rg#67]

**Comment by gregorypratt** [comment]
Yeah nice. Might expand on this idea and make use of moment.js...

**Comment by ckcollab** [comment]
Oh man, I knew there was some library that already did this better than I could :D

Yeah moment JS "Relative Time" seems perfect, is it OK to add dependencies to this project like that?

**Comment by gregorypratt** [comment]
I'm thinking create a `<rg-moment>` tag that essentially wraps the majority of the moment API - as far as moment dependency is concerned its fine to depend on a separate library.
